src/hooks/useDeleteTarget.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { useStatus, SUCCESS } from '@rootstrap/redux-tools';
+import { deleteTarget } from 'actions/targetActions';
+
+const useDeleteTarget = (setShowDeleteConfirmation, requestTargets, hideTargetForm) => {
+  const { status } = useStatus(deleteTarget);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (status === SUCCESS) {
+      setShowDeleteConfirmation(false);
+      requestTargets();
+      hideTargetForm();
+    }
```

Comment:
maybe you should return a function to reset the status? `createThunk(...).reset` If I recall correctly. You can check redux-tools readme,